README.md

```diff
@@ -49,16 +49,26 @@ snowdrop	    https://snowdrop.github.io/helm
 ## Usage
 
 Requirements:
-- Connected/logged to a Openshift cluster
+- Connected/logged to a Kubernetes/Openshift cluster
 - Have installed [the Helm command line](https://helm.sh/docs/intro/install/)
 
 ### spring-boot-example-app
 
-This chart deploys and exposes a Spring Boot application in Openshift.
-To install it, you need to execute the following command:
+This chart deploys and exposes a Spring Boot application in Kubernetes or Openshift.
```

Comment:
`in Kubernetes or Openshift.` --> `on Kubernetes or OpenShift.`

Remark: Openshift name is `OpenShift` and not `Openshift` ;-)

Reply:
Changed